assets/src/scss/layout/_footer.scss

```diff
@@ -78,7 +76,9 @@
 }
 
 .footer-social-media {
-  color: var(--footer_links_color, $grey-20);
+  _-- {
+    color: $grey-20;
```

Comment:
Technically putting this together with the `a` selector below, which sets the hover color, would be more straightforward. But I preserved it so we don't change the CSS specificity.